lab_6_pipeline/pipeline.py

```diff
@@ -44,37 +86,61 @@ def __init__(self, lemma: str = "", pos: str = "", tags: str = ""):
         """
         Initializes MorphologicalTokenDTO
         """
+        self.lemma = lemma
+        self.pos = pos
+        self.tags = tags
 
 
 class ConlluToken:
     """
     Representation of the CONLL-U Token
     """
 
-    def __init__(self, text: str):
+    def __init__(self, text: str, position: int = 0):
         """
         Initializes ConlluToken
         """
+        self._text = text
+        self.position = position
+        self._morphological_parameters = MorphologicalTokenDTO()
 
     def set_morphological_parameters(self, parameters: MorphologicalTokenDTO) -> None:
         """
         Stores the morphological parameters
         """
+        self._morphological_parameters = parameters
 
     def get_morphological_parameters(self) -> MorphologicalTokenDTO:
         """
         Returns morphological parameters from ConlluToken
         """
+        return self._morphological_parameters
 
     def get_conllu_text(self, include_morphological_tags: bool) -> str:
         """
         String representation of the token for conllu files
         """
+        position = str(self.position)
+        text = self._text
+        lemma = self._morphological_parameters.lemma
+        pos = self._morphological_parameters.pos
+        xpos = '_'
+        if include_morphological_tags and self._morphological_parameters.tags:
+            feats = self._morphological_parameters.tags
+        else:
+            feats = '_'
+        head = '0'
+        deprel = 'root'
+        deps = '_'
+        misc = '_'
```

Comment:
i would set these in return statement directly

Reply:
Done